src/components/document/document-content.tsx

```diff
@@ -200,8 +203,8 @@ export class DocumentContentComponent extends BaseComponent<IProps, IState> {
   }
 
   private renderSpacer = () => {
-    return !this.props.readOnly &&
-            <div className="spacer" onClick={this.handleClick} />;
+    const spacerClass = classNames({"spacer" : !this.props.readOnly}, {"playback-spacer": this.props.showPlayback});
```

Comment:
```suggestion
    const spacerClass = classNames({"spacer" : !this.props.readOnly, "playback-spacer": this.props.showPlayback});
```
FYI: you don't need two separate objects here.